pandas/core/indexes/base.py

```diff
@@ -696,12 +696,38 @@ def memory_usage(self, deep=False):
     @deprecate_kwarg(old_arg_name='n', new_arg_name='repeats')
     def repeat(self, repeats, *args, **kwargs):
         """
-        Repeat elements of an Index. Refer to `numpy.ndarray.repeat`
-        for more information about the `repeats` argument.
+        Repeat elements of an Index.
 
-        See also
+        Returns a new index where each element of the current index
+        is repeated consecutively a given number of times.
+
+        Parameters
+        ----------
+        repeats : int
+            The number of repetitions for each element.
+        kwargs : keyword, value pairs
```

Comment:
I think we have a bug in the `validate_docstrings.py` script, but I don't think we want to have type for `kwargs` as it's always a `dict of {str: anything}`. Also the name should be `**kwargs`, that's definitely a bug in the script, that has been reported before.

Not sure if I'm missing something, but why don't we have `*args` in the docstrings?

Reply:
\+1 for using `**kwargs` instead of `kwargs` (sphinx does not seem to complain about it, that was what I first feared)

Reply:
I'm now confused @datapythonista . Should we use `kwargs` or `**kwargs` as the argument name in the docstring? Because in the mailing list you said `kwargs` is more common.

Reply:
I'll answer on the mailing list. Let's go for `**kwargs`, and I am trying to fix the validation script: https://github.com/pandas-dev/pandas/pull/20072